src/main/java/no/uib/inf112/core/RoboRally.java

```diff
@@ -24,17 +23,19 @@
     //DO NOT PUT ASSET HERE!!! only this directory should be specified in the in the working directory
     //see https://github.com/inf112-v19/Blodtorstige-robotet/wiki/Run-with-IntelliJ
     public static final String FALLBACK_MAP_FILE_PATH = MAP_FOLDER + File.separatorChar + "test.tmx";
+    public static int numberOfPlayers = 3;
```

Comment:
This should probally be `final static`? (and if so remember to rename it to `NUMBER_OF_PLAYERS`)

Reply:
No, because the number of players is not constant, but i could add a parameter in the constructor so it makes more sense.

Reply:
I assume you then intend for this to be changed by the user then via UI. In that case it should not really exist as it is not referenced anymore after the create method. Instead maybe have an initalize method for setting the number of players in player handler (f.eks move it so it is a parameter of `generatePlayers(boolean)`.

Reply:
Maybe, but when we create a new game via the UI later, we would probably make a new playerhandler and not just generate new players? I can remove the variable, if that is the problem.

Reply:
Yeah but now we have a magic number :/ My original point was that this is used as a static final variable so it should be written as one

Reply:
Whatever, its not permanent anyway